openfisca_uk/variables/finance/benefit/family/housing_benefit.py

```diff
@@ -49,7 +49,7 @@ def formula(benunit, period, parameters):
         would_claim = benunit("claims_legacy_benefits", period) * (
             random(benunit) < parameters(period).benefit.housing_benefit.takeup
         )
-        return already_claiming + would_claim
+        return would_claim
```

Comment:
this moves some current claimants off it right? this sounds like the simplest way to model the correct take-up rate but i'd suggest noting it here and continuing to discuss in https://github.com/PSLmodels/openfisca-uk/issues/163 (if we expand to include HB) or a new issue whether we can prioritize keeping people on existing benefits and still get the right take-up.

Reply:
I agree this would be good to discuss further. Does seem like retaining claimed benefit attachment becomes less accurate the further away we get from 2018/19, given the UC rollout has changed significantly since then.